### PR TITLE
Clarify the purpose of the guides

### PR DESCRIPTION
**Pull request by mjankowski**

### CODE REVIEW

File: style/README.md [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+and not as an end in itself.
+
+Your time is valuable.
 Use [Hound] to automatically review your
 GitHub pull requests for
 style guide violations.
 
 [Hound]: https://houndci.com
 
+
```
Comment (mike-burns) [inline style/README.md]: Why is this newline here?

Comment (mjankowski) [inline style/README.md]: That's easily the best comment anyone could have left on this PR.

Everyone else - just stop yourselves now.  You are not going to do better than Mike.

File: style/README.md [hunk 2]
```diff
@@ -1,14 +1,22 @@
 Style
 =====
 
-A guide for programming in style.
+Delivering valuable features
+and working software
+are our ultimate goals.
 
+This style guide should be used
```
Comment (mcmire) [inline style/README.md]: I'm not sure I understand this sentence.

File: style/README.md [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
+in service of those goals,
+and not as an end in itself.
+
+Your time is valuable.
```
Comment (mcmire) [inline style/README.md]: This concept seems like it could be expanded.

**Comment by mcmire** [comment]
Didn't Jon write a blog post about why having a team style guide is important? Can't we borrow some ideas from that?

**Comment by jyurek** [comment]
I did! https://robots.thoughtbot.com/collaboration-is-other-people

**Comment by jferris** [comment]
We also had this post: https://robots.thoughtbot.com/why-does-style-matter

I think it would be worth pulling in some ideas from those.

**Comment by mike-burns** [comment]
Matt, do you now understand the purpose of the guides?

**Comment by tysongach** [comment]
:+1:

:clap: merge :clap: it :clap:

File: style/README.md (hunk 1 above)
Comment (sikachu) [inline style/README.md]: But ... should this empty line be removed?

File: style/README.md [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
 =====
 
-A guide for programming in style.
+Delivering valuable features
+and working software
+are our ultimate goals.
 
+This style guide should be used
+in service of those goals,
+and not as an end in itself.
```
Comment (jasonramirez) [inline style/README.md]: This last line doesn't really communicate anything to me that "should be used in service of those goals" doesn't already say.

**Pull request closed without merging**